src/View/View.php

```diff
@@ -655,13 +660,13 @@ public function element(string $name, array $data = [], array $options = []): st
             return $this->_renderElement($file, $data, $options);
         }
 
-        if (empty($options['ignoreMissing'])) {
-            [$plugin, $elementName] = $this->pluginSplit($name, $pluginCheck);
-            $paths = iterator_to_array($this->getElementPaths($plugin));
-            throw new MissingElementException([$name . $this->_ext, $elementName . $this->_ext], $paths);
+        if ($options['ignoreMissing']) {
+            return '';
```

Comment:
Do we need to test this option or is it tested by something else?

Reply:
It's tested separately. Earlier if this option was set it would get into the `_elementCache()` call and wasn't unset. Unneeded options are now cleaned up in `element()` itself before calling `_elementCache()`.